From: toddrme2178@gmail.com
Date: 2019-11-05
Subject: input: loosen certifi version dependency

Make the version dependency for certifi less strict. 
We don't use that exact version.

---

--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@
         'dill',
         'nose',
         'unittest2',
-        'certifi==2017.7.27.1',
+        'certifi>=2017.7.27.1',
         'requests',
         'lmdb',
         'redis'
